src/test/components/TrackContextMenu.test.js

```diff
@@ -96,6 +96,11 @@ describe('timeline/TrackContextMenu', function() {
       expect(container.firstChild).toMatchSnapshot();
     });
 
+    it('matches the snapshot of a global non-process track', () => {
+      const { container } = setupGlobalTrack(getScreenshotTrackProfile());
+      expect(container.firstChild).toMatchSnapshot();
+    });
+
```

Comment:
These test changes are not exactly related to this change, right?(It's related to track context menu, but not about showing PIDs) I'm okay with landing it with this PR but it would be better if we can split this to a new commit with it's own commit message.

Reply:
Since I also changed how menu items render when there are no PIDs, I added this snapshot to check that those won't break. So I'd say it is related. But I've moved that test into a separate commit, as you said.